src/autoyast-rnc/networking.rnc

```diff
@@ -266,9 +266,9 @@ routes =
 route =
   element (route | listentry) {
     destination &
-    element netmask { text } &	#overloaded
-    element device { text } &	#overloaded
-    gateway
```

Comment:
Allowed to avoid the declaration of `netmask`. `device` and `gateway`. So something like:
```xml
<route>
  <destination>default</destination>
  <device>eth0</device>
</route>
```